src/utils/components/SelectSnapshot/SelectSnapshotNamespace.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { Dispatch, FC, ReactElement, SetStateAction, useCallback, useState } from 'react';
+
+import { ProjectModel } from '@kubevirt-ui/kubevirt-api/console';
+import Loading from '@kubevirt-utils/components/Loading/Loading';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { ResourceLink } from '@openshift-console/dynamic-plugin-sdk';
+import { FormGroup, Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+
+type SelectSnapshotNamespaceProps = {
+  isDisabled?: boolean;
+  onChange: Dispatch<SetStateAction<string>>;
+  projectsLoaded: boolean;
+  projectsName: string[];
+  selectedProject: string;
+};
+
+export const FilterSnapshotSelect = (options: string[]) => {
```

Comment:
seems to me like its repeating with the file above

Reply:
Yes, I didn't notice. I'll do a common utils.